pkgs.nix

```diff
@@ -1,23 +1,21 @@
-{ sources ? import ./nix/sources.nix
-, config ? { }
-, system ? builtins.currentSystem
-,
-}:
-let
+{
+  sources ? import ./nix/sources.nix,
+  config ? {},
+  system ? builtins.currentSystem,
+}: let
   nixpkgs = sources.nixpkgs;
   rust_overlay = import (sources.rust_overlay);
 
   overlays = [
     (self: super: {
-      criterion = self.callPackage (import ./criterion.nix) { criterion = super.criterion; };
-      solana-source = self.callPackage (import ./solana-source.nix) { };
-      solana-platform-tools = self.callPackage (import ./solana-platform-tools.nix) { };
-      solana-rust = self.callPackage (import ./solana-rust.nix) { };
-      solana-cli = self.callPackage (import ./solana-cli.nix) { };
-      anchor-cli = self.callPackage (import ./anchor-cli.nix) { };
-    }
-    )
+      solana-source = self.callPackage (import ./solana-source.nix) {};
+      solana-platform-tools =
+        self.callPackage (import ./solana-platform-tools.nix) {};
```

Comment:
in flake.nix, there's a custom crane override for these. Is that not needed here?

Reply:
`crane` is just a way for nix to orchestrate the build for rust crates, but in this case platform-tools is just a set of binaries as we don't built them from source in the derivation, hence why it's not needed.